.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,34 @@
+name: Maintenance Mode
+
+on:
+  push:
+    branches:
+      - nick/2315-maintenance-banner
+  # workflow_dispatch:
+  #   inputs:
+  #     active:
+  #       description: "Active outage? (true/false)"
+  #       required: true
+  #       default: "true"
+  #     message:
+  #       description: "Message to display"
+  #       required: true
+
+jobs:
+  pushMaintenanceBlob:
```

Comment:
What's the workflow for removing the maintenance banner?

Reply:
Just run the job again with `active: false`!

Reply:
Oh perfect! LGTM in that case